weaviate_cli/commands/create.py

```diff
@@ -92,7 +92,7 @@ def create() -> None:
     "--vectorizer",
     default=CreateCollectionDefaults.vectorizer,
     type=click.Choice(
-        ["contextionary", "transformers", "openai", "ollama", "cohere", "jinaai"]
+        ["contextionary", "transformers", "openai", "ollama", "cohere", "jinaai", "text2vec_weaviate"]
```

Comment:
maybe we could simplify the naming as with the other vectorizers and leave it as weaviate only?

Reply:
Yes. I have changed it as suggested by you